tests/test_magnetic_fields.py

```diff
@@ -1244,6 +1384,55 @@ def test_omnigenous_field_change_resolution_B(self):
         np.testing.assert_allclose(B_half_lowres, B_half_highres, rtol=3e-3)
         np.testing.assert_allclose(B_lcfs_lowres, B_lcfs_highres, rtol=4e-3)
 
+    @pytest.mark.unit
+    def test_solve_current_potential_warnings_and_errors(self):
+        """Test solve current potential warnings/errors."""
+        field = FourierCurrentPotentialField(
+            I=0,
+            G=1,
+            sym_Phi="sin",
+        )
```

Comment:
You love to spread things that should be a single line of code over as many lines as possible...

Reply:
I dont do this, my IDE does it automatically

Reply:
It will automatically format it like this only if you add the extra comma after the last argument. If you remove that unnecessary comma it should format it onto a single line

Reply:
yea I have made that change now, thanks. Just sometimes forget to